go.mod

```diff
@@ -255,6 +255,41 @@ require (
 	k8s.io/utils v0.0.0-20211116205334-6203023598ed // indirect
 	launchpad.net/gocheck v0.0.0-20140225173054-000000000087 // indirect
 	sigs.k8s.io/json v0.0.0-20211020170558-c049b76a60c6 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
+	sigs.k8s.io/yaml v1.2.0 // indirect
+)
+
+require (
+	k8s.io/cli-runtime v0.23.3
+	k8s.io/kubectl v0.23.3
```

Comment:
I don't see these being used anywhere. Was adding them on purpose?

Reply:
not sure why it got messed up during the rebase. cleaned it now